src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -316,12 +316,16 @@ sparse sequences.
-| Instructions    | RECOMMENDED. Text of the instructions given to participants before the scan. This is especially important in context of resting state fMRI and distinguishing between eyes open and eyes closed paradigms. |
-| TaskDescription | RECOMMENDED. Longer description of the task.                                                                                                                                                               |
-| CogAtlasID      | RECOMMENDED. URL of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) Task term.                                                                                                        |
-| CogPOID         | RECOMMENDED. URL of the corresponding [CogPO](http://www.cogpo.org/) term.                                                                                                                                 |
+| Field name                  | Definition                                                                                                                                                                                                 |
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Instructions                | RECOMMENDED. Text of the instructions given to participants before the scan. This is especially important in context of resting state fMRI and distinguishing between eyes open and eyes closed paradigms. |
+| TaskDescription             | RECOMMENDED. Longer description of the task.                                                                                                                                                               |
+| CogAtlasID                  | RECOMMENDED. URL of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) Task term.                                                                                                        |
+| CogPOID                     | RECOMMENDED. URL of the corresponding [CogPO](http://www.cogpo.org/) term.                                                                                                                                 |
+| StimuliPresentationOS       | RECOMMENDED. Operating system used to run the stimuli presentation software.                                                                                                                               |
+| StimuliPresentationSoftware | RECOMMENDED. Name of the software that was used to present the stimuli ideally with its [Research Ressource Identifier](https://scicrunch.org/resources)                                                   |
```

Comment:
should we give a more explicit example? E..g. that the specification is RRID:SCR_002521 and that it links to https://scicrunch.org/resolver/SCR_002521?

Reply:
please add a comma like this `... used to present the stimuli, ideally with its ...`

Reply:
> should we give a more explicit example? E..g. that the specification is RRID:SCR_002521 and that it links to https://scicrunch.org/resolver/SCR_002521?

Should I put the example in the table and / or in the sample JSON for that section?

Reply:
Please try out and pick whatever place you feel that it fits (and looks) the best.

Reply:
@robertoostenveld I have added some RRID examples and also extended the json examples. 

I tried to vary the names / gender / origin of the fictiitious github repo owners, though now that I think about it maybe we should put fictitious link to zenodo or more "stable" repos, to make sure the specs "lead" by example.